roberta/ev3.py

```diff
@@ -613,7 +613,7 @@ def getInfraredSensorSeek(self, port):
         s = self.cfg['sensors'][port]
         if s.mode != 'IR-SEEK':
             s.mode = 'IR-SEEK'
-        return self.scaledValue(s)
+        return (s.value(0), s.value(1), s.value(2), s.value(3), s.value(4), s.value(5), s.value(6), s.value(7))
 
```

Comment:
Shouldn't we apply scaling to each value?

Reply:
Ah, I just did it similarily to [getColorSensorRgb](https://github.com/boonto/robertalab-ev3dev/blob/8e98c02657fdc6e38133bce89a3efcbd502bced8/roberta/ev3.py#L604-L608).
I can add a new scaling function that handles the arrays.